mobile/src/main/res/values/common-styles.xml

```diff
@@ -7,15 +7,25 @@
   </style>
 
   <style name="TabStrip">
-    <item name="android:textColor">@color/show_details_tabs_text_default</item>
-    <item name="pstsTextColorSelected">@color/show_details_tabs_text_selected</item>
-    <item name="pstsIndicatorHeight">@dimen/show_details_tabs_indicator_height</item>
-    <item name="pstsIndicatorColor">@color/show_details_tabs_indicator</item>
-    <item name="pstsShouldExpand">true</item>
+    <item name="tabLayoutId">@layout/view_tab</item>
+    <item name="tabsLeftPadding">0dp</item>
+    <item name="indicatorColor">@color/tab_strip_indicator</item>
+    <item name="indicatorHeight">@dimen/tab_strip_indicator_height</item>
```

Comment:
Hey @ataulm the library looks great! I think is great you started from scratch a brand new library base on this one. You cleaned it up and organised the code so much better (there was not much organisation as I continue from the old tabstrip library. Maybe I should have spent more time on that as well :confused: ).

Great idea the tabLayoutId (I think you commented it at some point), TabSetterUpper and the new approach for shouldExpand.

Downsides.... not many for this approach, that at the end is focus on simplifying everything on the library side and leave hard work for the devs. 
I could say that for not experienced developers it will be more difficult to customise, because we loose some attr for underline , dividers and paddingMiddle.